pip/flatpak-pip-generator

```diff
@@ -311,9 +311,8 @@ for package in packages:
     else:
         modules.append(module)
 
-if requirements_file == '/tmp/flatpak-pip-generator-requirements.txt':
-    if os.path.exists(requirements_file):
-        os.remove(requirements_file)
+if not opts.requirements_file and os.path.exists(requirements_file):
+    os.remove(requirements_file)
```

Comment:
This is a race condition. Just always call `os.remove()` and handle it failing as normal.